.github/workflows/markdown-lint.yml

```
@@ -23,34 +23,51 @@ jobs:
+            local line_end="$3"
+            printf "::error file=%s,line=%s,endLine=%s,title=VALIDATION_ERROR::ERROR %s\n" \
+              "${file_stub}" "${line_start}" "${line_end}" "${msg}"
+          }
+          command -v npm >/dev/null || { print_error "npm installation failed." 26 27 ; exit 126; }
+          yamllint --version || { print_error "Yamllint installation failed." 26 27 ; exit 126; }
 
       - name: Lint Workflow YAML
+        if: ${{ success() }}
+        env:
+          YAML_ARGS: "${{ vars.YAML_ARGS }}"
         run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2086:info:2:10: Double quote to prevent globbing and word splitting

(shellcheck)